### DISCUSSION TITLE
Compiling on ARCHER2

### DISCUSSION THREAD

**Discussion by AndrewILWilliams** [ExeClim/Isca#209] [Q&A] [answered]
Hi there!

I'm just wondering if anyone has tried/successfully set up Isca on the ARCHER2 HPC yet? If not then I can put in a support request to the help desk, but this will be my first time using Isca so I thought it best to ask just in case people have any tips for compiling. Is it *actually* just as simple as setting the environmental variables and then installing the python package? Or am I missing something?

Thanks very much, and enjoy the weekend! :)

**Comment by gkvallis** [comment, marked as answer]
I do not think anyone has actually run it on Archer2, at least to my knowledge. It has been run on similar supercomputers, for example at PNNL (a DoE lab in the USA) and on Isambard (an ARM supercomputer).  But if you haven't run Isca before it might be easier to try on a local machine first; Neil Lewis in AOPP has set Isca up on some Oxford machines and might be able to help if you contact him directly. 

It would be good to have a working Archer configuration though, so good luck with that and let us know how it goes.

**Reply by AndrewILWilliams** [reply]
Great, I'll ask Neil :) I'll also put in a PR if I get it running on ARCHER.
